<?xml version="1.0" encoding="utf-8"?>
<launch>
  <arg name="odom_topic" value="/state_ukf/odom" />
  <arg name="goal_topic" value="/goal" />
  <arg name="waypoints_path" default="/home/kyle/ros_ws/faster-planner/src/krrt-planner/decision/state_machine/launch/RCwoods.txt" />

  <node pkg="state_machine" type="pub_goal_node" name="pub_goal_node" output="screen" >
    <remap from="/odom_topic" to="$(arg odom_topic)"/>
    <remap from="/goal" to="$(arg goal_topic)"/>
    <param name="waypoints_path" value="$(arg waypoints_path)"/>
    <param name="avg_vel" value="3.1" type="double"/>
    
  </node>
  
</launch>
